gallery/src/styles/NxPageTitle/NxPageTitlePage.tsx

```diff
@@ -48,13 +52,6 @@ const NxPageTitlePage = () =>
               <code className="nx-code">&lt;h1&gt;</code>.
             </td>
           </tr>
-          <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-page-title__page-icon</code></td>
-            <td className="nx-cell">Element</td>
-            <td className="nx-cell">
-              Class for an icon that appears to the left of the page title.
-            </td>
-          </tr>
           <tr className="nx-table-row">
```

Comment:
Needs a row for the subtitle class

Reply:
> Needs a row for the subtitle class

Fixed and added convenience class: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/8fd0ec709b78832a8333fd0668801d4e1ff617be